classes/shaman.lua

```diff
@@ -50,6 +146,26 @@ local function registerClass(self)
 
     if self.IsSoD() then
 
+        --Initializing Rolling Thunder handler
+        if (not rollingThunderHandler.initialized) then
+            rollingThunderHandler:init();
+        end
+        
+        --Deactivating Rolling Thunder on wrists rune change and activating on login if stack count is right, not refreshing overlay is currently active
+        local RTRuneUpdateTracker = CreateFrame("FRAME");
+            RTRuneUpdateTracker:RegisterEvent("SPELLS_CHANGED");
+            RTRuneUpdateTracker:SetScript("OnEvent", function(self)
+                local RollingThunderEquipped = C_Engraving and SAO:IsSpellLearned(432056);
+                if not RollingThunderEquipped then
+                    rollingThunderHandler:deactivate();
+                else
+                    local _, _, stacks = AuraUtil.FindAuraByName(GetSpellInfo(324), "player")

```

Comment:
Relying on `FindAuraByName` is usually bad practice, but I guess this is acceptable considering that:
1. this check is done after `SPELLS_CHANGED`, which I believe is not triggered very often
2. the extra code required to track more _optimally_ the same info would be way too complex